guides/common/modules/proc_installing-the-dns-infoblox-module.adoc

```diff
@@ -13,13 +13,15 @@ DNS records are managed only in the default DNS view, it's not possible to speci
 ----
 # {foreman-installer} --enable-foreman-proxy-plugin-dns-infoblox \
 --foreman-proxy-dns true \
+--foreman-proxy-dns-managed false \
```

Comment:
This isn't needed since https://github.com/theforeman/puppet-foreman_proxy/commit/54067fe5ac742000d3c2cdb8505d8f191383aeb7, which was Foreman 2.0.
```suggestion
```